spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb

```diff
@@ -295,9 +295,13 @@ def assert_specific_security_group
   end
 
   def assert_specific_flavor
-    @flavor = ManageIQ::Providers::Azure::CloudManager::Flavor.where(:name => "Basic_A0").first
+    @flavor_not_found = ManageIQ::Providers::Azure::CloudManager::Flavor.where(:name => "Basic_A0").first
+    expect(@flavor_not_found).to eq(nil)
+
+    @flavor = ManageIQ::Providers::Azure::CloudManager::Flavor.where(:name => "basic_ao").first
```

Comment:
This test is not testing the same thing.  The first one is testing A<zero>, the second is testing A<letter-o>